capstone-ui/src/components/Settings/Settings.jsx

```diff
@@ -133,10 +131,20 @@ export default function Settings({ username, token, profile, isRegisterView }) {
               </button>
             </div>
           )}
-          {recommendations && (
-            <span className="recommendations-heading">Recommendations</span>
+          {isFetchingRecs && (
+            <div className="loading-recs-wrapper">
+              {<LoadingSpinner></LoadingSpinner>}
+            </div>
+          )}
+          {!isFetchingRecs && recommendations && (
```

Comment:
recommendations != null &&

Reply:
Fixed in the play feature PR will chose that change when merging